makefile

```
@@ -0,0 +1,21 @@
+CLANG_FORMAT=node_modules/clang-format/bin/darwin_x64/clang-format --style=Google
```

Could you set this back to 
```suggestion
CLANG_FORMAT=node_modules/clang-format/bin/linux_x64/clang-format --style=Google
```
Because I'm planning on running this in cloud shell, which I think is based on Ubuntu.

---

Looks like I was able to apply this directly in the web UI.